source/reference/api/start.txt

```diff
@@ -362,23 +362,20 @@ If the ``start`` request is successful, ``mongosync`` enters the
 Pre-Split Chunks
 ~~~~~~~~~~~~~~~~
 
-.. versionadded:: 1.1
-
-When ``mongosync`` syncs to a sharded cluster, it pre-splits chunks for
-sharded collections on the destination cluster.  This is supported in the
-following configurations:
+When ``mongosync`` syncs to a sharded destination cluster, it pre-splits chunks 
+for sharded collections on the destination cluster. For each sharded collection, 
+``mongosync`` creates twice as many chunks as there are shards in the 
+destination cluster. The following configurations pre-split chunks:
 
-* Sync from a replica set to a sharded cluster.
+- Sync from a replica set to a sharded cluster.
 
-* Sync between sharded clusters that differ in the number of shards.
+- Sync between sharded clusters.
```

Comment:
I'd say that these bullets are de facto covered by us saying "When `mongosync` syncs to a sharded destination cluster" and that we can remove them, but I'll leave the final decision up to you.

Reply:
Ah gotcha ! I'll opt for removing then :)